org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/CollectionLiterals.java

```diff
@@ -154,6 +156,8 @@
 	 * Creates an empty mutable {@link LinkedList} instance.
 	 * 
 	 * @return a new {@link LinkedList}
+	 * 
```

Comment:
The if is at beta so I am not sure if we need the since tags

Reply:
You commented collection literal, but you meant MutableAnnotationTypeElementDeclaration?
Yes, it is beta, but the since tag is still useful, don't you think? The Beta annotation marks this interface as subject to change, so nobody could complain when a method is removed. But yet users will know when methods are introduced, and they are actually API. I think it is useful to leave the tags, but if required I will remove them.